### ISSUE TITLE
Error: spawn /tmp EACCES

### ISSUE THREAD

**Issue by atulnag** [AntonelliLab/raxmlGUI#228]
Autogenerated report:
```json
{
  "errno": -13,
  "code": "EACCES",
  "syscall": "spawn /tmp/.mount_raxmlG5uEsDZ/resources/static/bin/modeltest-ng",
  "path": "/tmp/.mount_raxmlG5uEsDZ/resources/static/bin/modeltest-ng",
  "spawnargs": [
    "-d",
    "aa",
    "-i",
    "/home/atul/Documents/PhDWork/ARF/Version3/Phylogeny/CcARF_jalview.fa",
    "-o",
    "/home/atul/Documents/PhDWork/ARF/Version3/Phylogeny/RAxML_GUI_ModelTest_CcARF_jalview",
    "--force",
    "-p",
    2
  ],
  "name": "Error",
  "message": "spawn /tmp/.mount_raxmlG5uEsDZ/resources/static/bin/modeltest-ng EACCES",
  "stack": "Error: spawn /tmp/.mount_raxmlG5uEsDZ/resources/static/bin/modeltest-ng EACCES"
}
```

Active state:
```json
{
  "command": "modeltest-ng -d aa -i /home/atul/Documents/PhDWork/ARF/Version3/Phylogeny/CcARF_jalview.fa -o /home/atul/Documents/PhDWork/ARF/Version3/Phylogeny/RAxML_GUI_ModelTest_CcARF_jalview --force -p 2",
  "stdout": ""
}
```

---

Process: renderer
raxmlGUI 2.0.6
Electron 13.5.1
linux 5.11.0-43-generic
Locale: en-GB

**Comment by danieledler** [comment]
Hi @atulnag, thanks for reporting the issue!

Maybe it is also related to issue #226, does the [2.0.5](https://github.com/AntonelliLab/raxmlGUI/releases/tag/v2.0.5) version works for you too?

**Comment by jtklein** [comment]
Should be fixed with v 2.0.7
